helper/repository.php

```diff
@@ -390,6 +390,10 @@ public function getFilteredPlugins(
             $plugins[$i]['compatible'] = $this->cleanCompat($plugins[$i]['compatible']);
             $plugins[$i]['types']      = $this->listtypes($plugins[$i]['type']);
 
+            if(in_array($plugins[$i]['securitywarning'],$this->securitywarning)){
+                $plugins[$i]['securitywarning'] = $this->getLang('security_'.$plugins[$i]['securitywarning']);
+            }
```

Comment:
So when this function always returns the resolved string, I guess resolving at other places like syntax/entry/_showWarnings() is redundant and should be removed?